bin/oneoff/data_fix/remove_admin_from_non_sso

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+
+#
+# Script to remove admin privileges from accounts without Google SSO
+#
+
+require_relative '../../../dashboard/config/environment'
+
+File.readlines(ARGV[0]).each do |line|
+  id = line.to_i
+  user = User.find_by_id(id)
+  if user.nil?
+    puts("user #{id} - ERROR - does not exist")
+    next
+  end
+
+  unless user.admin
+    puts("user #{id} - ERROR - is not admin")
+    next
+  end
+
+  if user.
+      authentication_options.
+      where(credential_type: "google_oauth2").
+      where("email LIKE ?", '%@code.org').
```

Comment:
I didn't see any mention in the ticket that we should only allow users who specifically have code.org SSO credentials, but that seems important so I added it.

Reply:
Awesome!  thanks for including.